uint32  class_id
string  class_name
Rect    box
float32 confidence